pkg/api/types.go

```diff
@@ -515,8 +524,8 @@ type PodSpec struct {
 // PodStatus represents information about the status of a pod. Status may trail the actual
 // state of a system.
 type PodStatus struct {
-	Phase PodPhase `json:"phase,omitempty"`
-
+	Phase      PodPhase       `json:"phase,omitempty"`
+	Conditions []PodCondition `json:"Condition,omitempty"`
```

Comment:
I would like to add PodCondition now. I'm ok we initially omit some fields, such as LastTransitionTime, Reason, and Message.